index.js

```diff
@@ -164,6 +164,9 @@ function SpotifyResolve(createOpts) {
       if (error) {
         done(error);
       }
+      else if (response.statusCode === 401) {
+        done(new Error(response.statusCode), response);
```

Comment:
This is the right idea, but it would be if there were a little more to the error message other than the code. Perhaps `'Authorization error: Status code 401 received.'`.